src/renderers/WebGLRenderer.js

```diff
@@ -2261,6 +2261,33 @@ class WebGLRenderer {
 
 		};
 
+		this.canReadPixelFormat = function ( textureFormat ) {
```

Comment:
I can see how these methods are helpful.

How about we move the code to `renderer.capabilities` similar to `getMaxAnisotropy()`? On app level, you would do:
```js
if ( renderer.capabilities.canReadTextureType( THREE.FloatType ) ) {
```

Reply:
Done. I had to instantiate `WebGLUtils` in `WebGLCapabilities`, though.